### PR TITLE
Use forked opentelemetry js

### PR DESCRIPTION
**Pull request by RafalSumislawski**

### CODE REVIEW

**Comment by zhrebicek** [comment]
Looks ok apart fro failing shellcheck

**Review by zhrebicek — APPROVED** [review]
LGTM, do not you need that new `il` region?

File: .github/workflows/publish-nodejs.yml [hunk 1]
```diff
@@ -15,26 +15,45 @@ jobs:
...
+    - name: Build opentelemetry-instrumentation-aws-lambda
+      working-directory: ${{ env.OPENTELEMETRY_JS_CONTRIB_PATH }}/plugins/node/opentelemetry-instrumentation-aws-lambda
       run: npm install --ignore-scripts && npm pack --ignore-scripts
-    - name: Build instrumentation-aws-sdk
-      working-directory: ./opentelemetry-js-contrib/plugins/node/opentelemetry-instrumentation-aws-sdk
+    - name: Build opentelemetry-instrumentation-aws-sdk
+      working-directory: ${{ env.OPENTELEMETRY_JS_CONTRIB_PATH }}/plugins/node/opentelemetry-instrumentation-aws-sdk
       run: npm install --ignore-scripts && npm pack --ignore-scripts
+    - name: Prepare opentelemetry-instrumentation
+      working-directory: ${{ env.OPENTELEMETRY_JS_PATH }}
```
Comment (lukaszwojtowcoralogix) [inline .github/workflows/publish-nodejs.yml:45]: Sometimes working-directory is OPENTELEMETRY_JS_PATH and sometimes ...CONTRIB_PATH. Is this correct?

**Review by lukaszwojtowcoralogix — APPROVED** [review]

File: .github/workflows/publish-nodejs.yml (hunk 1 above)
Comment (RafalSumislawski) [reply .github/workflows/publish-nodejs.yml:45]: Yes. That's correct. I'm building 3 libraries coming from 2 different repositories (cloned into these two directories).

**Comment by RafalSumislawski** [comment]
> LGTM, do not you need that new `il` region?

Right :+1:. Added it.

**Pull request merged**